glean-core/src/metrics/labeled.rs

```diff
@@ -127,7 +127,7 @@ where
     ///
     /// If the requested label is in the list of allowed labels, it is returned.
     /// Otherwise the `OTHER_LABEL` is returned.
-    fn static_label<'a>(&mut self, label: &'a str) -> &'a str {
+    fn static_label<'a>(&self, label: &'a str) -> &'a str {
```

Comment:
Oh, did this come up through a lint now that these don't have to me mutable anymore?

Reply:
Not through a lint, before the changes suggested by @mdboom I noticed it.